test/commands/hset.js

```diff
@@ -24,4 +24,20 @@ describe('hset', () => {
           email: 'bruce@wayne.enterprises',
         })
       ));
+
+  it('should let you set multiple hash map keys and values in a single command', () => {
+    const hash = { id: '2', email: 'bruce@wayne.enterprises' };
+    return redis
+      .hset('user:2', 'id', '2', 'email', 'bruce@wayne.enterprises')
+      .then(status => expect(status).toBe(2))
+      .then(() => expect(redis.data.get('user:2')).toEqual(hash));
+  });
+
+  it('should let you set multiple hash map keys and values with an object', () => {
+    const hash = { id: '3', email: 'bruce@wayne.enterprises' };
+    return redis
+      .hset('user:3', hash)
+      .then(status => expect(status).toBe(1))
+      .then(() => expect(redis.data.get('user:3')).toEqual(hash));
+  });
```

Comment:
Note these are both proper and equivelant ways to call hget (after redisv4) and should both have tests:
  redis.hset('mykey', {'field1': 'fieldval1', 'field2', 'fieldval2'})
  redis.hset('mykey', 'field1', 'fieldval1', 'field2', 'fieldval2')

eg, a subsequent call to redis.hgetall('mykey') should return all the data for the key